atom/renderer/api/atom_api_web_frame.cc

```diff
@@ -530,12 +522,7 @@ void Initialize(v8::Local<v8::Object> exports,
   dict.SetMethod("executeJavaScript", &ExecuteJavaScript);
   dict.SetMethod("executeJavaScriptInIsolatedWorld",
                  &ExecuteJavaScriptInIsolatedWorld);
-  dict.SetMethod("setIsolatedWorldSecurityOrigin",
-                 &SetIsolatedWorldSecurityOrigin);
-  dict.SetMethod("setIsolatedWorldContentSecurityPolicy",
-                 &SetIsolatedWorldContentSecurityPolicy);
-  dict.SetMethod("setIsolatedWorldHumanReadableName",
-                 &SetIsolatedWorldHumanReadableName);
+  dict.SetMethod("setIsolatedWorldInfo", &SetIsolatedWorldInfo);
```

Comment:
Did we update the docs for this? We should probably continue to support the old API as deprecated for 5-0-x

Reply:
@nornagon There is a separate PR to deprecate the current API in `5-0-x`

Reply:
https://github.com/electron/electron/pull/16801